src/access_list.rs

```diff
@@ -65,67 +71,70 @@ impl AccessListInspector {
     /// top-level call.
     ///
     /// Those include caller, callee and precompiles.
-    fn collect_excluded_addresses<DB: Database>(&mut self, context: &EvmContext<DB>) {
-        let from = context.env.tx.caller;
-        let to = if let TxKind::Call(to) = context.env.tx.transact_to {
+    fn collect_excluded_addresses<CTX: ContextTrait<Journal: JournalExt>>(
+        &mut self,
+        context: &CTX,
+    ) {
+        let from = context.tx().caller();
+        let to = if let TxKind::Call(to) = context.tx().kind() {
             to
         } else {
             // We need to exclude the created address if this is a CREATE frame.
             //
             // This assumes that caller has already been loaded but nonce was not increased yet.
-            let nonce = context.journaled_state.account(from).info.nonce;
+            let nonce = context.journal_ref().evm_state().get(&from).unwrap().info.nonce;
```

Comment:
nit, can we drop the _ref() suffix here? should just be journal and journal_mut

Reply:
In most cases a mutable journal is needed, there was few cases were ref is needed so this fn was added

Reply:
I see, then I think it should remain like that for legacy reasons